pkg/app/app.go

```diff
@@ -164,9 +165,36 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval, err := time.ParseDuration(a.config.Broker.RefreshInterval)
+	a.log.Debug("RefreshInterval: %v", interval.String())
+	if err != nil {
+		a.log.Error(err.Error())
+		a.log.Error("Not using a refresh interval")
```

Comment:
We should print the value `a.config.Broker.RefreshInterval` in the error. If I get an error I want to know what value was entered.